multichain-testing/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -103,21 +103,44 @@ const toOracleOfferId = (idx: number) => `oracle${idx + 1}-accept`;
 
 test.serial('oracles accept', async t => {
   const { oracleWds, retryUntilCondition, vstorageClient, wallets } = t.context;
+  const brands = await vstorageClient.queryData('published.agoricNames.brand');
+  const { Invitation } = Object.fromEntries(brands);
 
-  const instances = await vstorageClient.queryData(
-    'published.agoricNames.instance',
-  );
-  const instance = fromEntries(instances)[contractName];
+  // TODO export/import INVITATION_MAKERS_DESC
+  const ORACLE_INV_DESC = 'oracle operator invitation';
```

Comment:
for the reason above I think this shouldn't have ALL_CAPS casing. defining the value at all is just a convenience in this function to not have to repeat it.
```

```suggestion
  const description = 'oracle operator invitation';
```